clients/extension/src/background/index.ts

```diff
@@ -590,14 +577,15 @@ const handleRequest = (
           const [param] = params
 
           if (param?.chainId) {
-            const chainFromId = getChainByChainId(param.chainId)
-            const supportedChain = isSupportedChain(chainFromId)
-              ? chainFromId
+            const supportedChain = isSupportedChain(
+              getChainByChainId(param.chainId)
```

Comment:
I don't understand this change either , the previous version , you all `getChainByChainId` once , 

the current one , you call it twice back to back , I think the previous one is much better

Reply:
Done